examples/quickstart_fastai/README.md

```diff
@@ -0,0 +1,67 @@
+# Flower Example using fast.ai
+
+This introductory example to Flower uses fast.ai, but deep knowledge of fast.ai is not necessarily required to run the example. However, it will help you understand how to adapt Flower to your use case.
```

Comment:
```suggestion
This introductory example to Flower uses [fast.ai](https://www.fast.ai/), but deep knowledge of fast.ai is not necessarily required to run the example. However, it will help you understand how to adapt Flower to your use case.
```